client/src/App.css

```diff
@@ -1 +1,17 @@
+body {
+    background-color: #1C468E;
+}
 
+form {
+    display: flex; 
+    align-items: center;
+    flex-direction: column; 
+    justify-content: center; 
+    margin-top: 10px; 
+}
+
+.edit {
+    display: flex; 
+    flex-direction: row;
+    margin-top: 20px;
+}
```

Comment:
1. I would suggest you use MUI `<Box>` and style it inline using the `sx` prop. This can surround (or be inside) the `<form>` HTML element and you can switch out the `<div>` for a `<Box>` in `EditContactForm.jsx`. Generally we won't style the `<form>` HTML element and it's just a kind of "invisible" container simply for form data management purposes. Another example: check out how I use a hidden `<input>` for selecting photo files in my app at `client/src/pages/logged-in-pages/Location/ImageCarousel/index.tsx` - this element is not actually visible to the user and is only there to manage the action of triggering showing the file upload dialog and forwarding the file data to my JS code for further processing.
2. If you are going to use traditional CSS files like this, always keep it as close to the code where it's used as possible. Meaning, I would suggest you move these 2 styles (`form` and `.edit`) into a .css file that sits in the `components` folder and is imported in `EditContactForm.jsx` instead of `App.jsx` so other devs won't have to go hunting for where this CSS is used.